website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -142,3 +142,50 @@ more information on the initial configuration of a BigQuery OAuth connection in
 As an end user, if your organization has set up BigQuery OAuth, you can link a project with your personal BigQuery account in your personal Profile in dbt Cloud, like so:
 
 <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/bq_oauth/bq_oauth_as_user.gif" title="Link Button in dbt Cloud Credentials Screen" />
+
+## Connecting to Databricks
+
+### ODBC
+
+dbt Cloud supports connecting to Databricks using
+[a Cluster](https://docs.databricks.com/clusters/index.html) or 
+[a SQL Endpoint](https://docs.databricks.com/sql/admin/sql-endpoints.html).
+Depending on how you connect to Databricks, either one of the `Cluster` or
+`Endpoint` configurations must be provided, but setting _both_ values is not
+allowed.
+
+The following fields are available when creating a Databricks connection:
+
+| Field | Description | Examples |
+| ----- | ----------- | -------- |
+| Host Name | The hostname of the Databricks account to connect to | `avc-def1234ghi-9999.cloud.databricks.com` |
+| Port | The port to connect to Databricks for this connection | 443 |
+| Organization | Optional, only required for Azure Databricks | 0123456789 |
+| Cluster | The ID of the cluster to connect to (required if using a cluster) | 1234-567890-abc12345 |
+| Endpoint | The ID of the endpoint to connect to (required if using Databricks SQL Analytics) | 0123456789 |
+| User | Optional | dbt_cloud_user |
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/databricks-connection.png" title="Configuring a Databricks connection"/>
+
+## Connecting to Apache Spark
+
+### HTTP and Thrift
+
+dbt Cloud supports connecting to an Apache Spark cluster using the HTTP method
+or the Thrift method.
```

Comment:
The inclusion of Databricks-relevant properties below (e.g. `Organization`) may be confusing, so you might want to include a note to the effect of:

```suggestion
dbt Cloud supports connecting to an Apache Spark cluster using the HTTP method
or the Thrift method. Note that, while the HTTP method can be used to connect to an all-purpose Databricks cluster, the ODBC method is recommended for all Databricks connections.
```